getting_started/basic_walkthrough.adoc

```diff
@@ -103,27 +107,26 @@ MongoDB pod then starts up and displays its newly-assigned IP address.
 
 In this section, you will configure a GitHub webhook to automatically trigger a
 rebuild of your application whenever you push code changes to your forked
-repository.
+repository. This involves adding the Github webhook URL from your application
+into your Github repository. You obtain this webhook from these locations:
```

Comment:
My reasoning was that when I tested it, I didn't know there was a second location to obtain the webhook URL (until I later scrolled down past the screenshot, I thought it was only obtainable from the post-creation page). So I ended up clicking back a couple of times to try and find the post-creation screen on my browser. It's not huge deal but it does save a minute of time knowing that the URL is available in multiple locations, which is why I think it's important to explicitly say that.